content/docs/other-guides/kustomize.md

```diff
@@ -0,0 +1,249 @@
+installation instructions based on your deployment scenario.
+
+The deployment process consists of three steps, _init_, _generate_, and 
+_apply_, so that you can modify your configuration before actually deploying 
+Kubeflow.
+
+* `kfctl init` - one time set up.
+* `kfctl generate` - creates config files defining the various resources.
+* `kfctl apply` - creates or updates the resources.
+
+### Specifying a configuration file when initializing your deployment
+
+When you install Kubeflow, the deployment process uses one of a few possible
+YAML configuration files to determine the configuration.
+
+Typically, you specify the configuration file with a `--config` parameter
+when you run `kfctl init`. For example:
+
+```
+export KFAPP="<your choice of application directory name>"
+export CONFIG="https://raw.githubusercontent.com/kubeflow/kubeflow/{{% kf-latest-version %}}/bootstrap/config/kfctl_existing_arrikto.yaml"
+kfctl init ${KFAPP} --config=${CONFIG} -V
+```
+
+*For details of the above deployment, see the guide to deployment using the 
+[kfctl_existing_arrikto configuration](/docs/started/k8s/kfctl-existing-arrikto/).*
+
+Some deployment processes use a default config file and you don't need to
+add the `--config` argument. For example, the 
+Google Cloud Platform (GCP) initialization command looks like this:
+
+```
+export PROJECT="<your GCP project ID>"
+export KFAPP="<your choice of application directory name>"
+kfctl init ${KFAPP} --platform gcp --project ${PROJECT}
+```
+
+*For details of the above deployment, see the guide to deployment 
+[on GCP using the CLI](/docs/gke/deploy/deploy-cli/).*
+
+### Your Kubeflow directory layout
+
+Your Kubeflow app directory is the directory where you choose to store 
+your Kubeflow configurations during deployment. This guide refers to the
+directory as `${KFAPP}`. The directory contains the  following files and 
+directories:
+
+* **app.yaml** stores your primary Kubeflow configuration in the form of a
+  `KfDef` Kubernetes object.
+
+  * The values are set when you run `kfctl init`.
+  * The values are snapshotted inside `app.yaml` to make your app 
+    self contained.
+  * The YAML defines each Kubeflow application as a kustomize package.  
+
+* **[platform-name]_config** is a directory that contains 
+  configurations specific to your chosen platform or cloud provider. This 
+  directory may or may not be present, depending on your setup.
+
+  * The directory is created when you run `kfctl generate platform`.
+  * You can modify these configurations to customize your infrastructure.
+
+* **kustomize** is a directory that contains Kubeflow application manifests.
+  That is, the directory contains the kustomize packages for the Kubeflow 
+  applications that are included in your deployment.
+
+  * The directory is created when you run `kfctl generate`.
+  * You can customize the Kubernetes resources by modifying the manifests and 
```

Comment:
Actually I think the official recommendation is to modify parameters in KfDef, and then re-running `kfctl generate`.

Reply:
Ah I see. Thanks.